dwm.cpp

```diff
@@ -1473,6 +1478,47 @@ setlayout(const Arg *arg)
 		drawbar(selmon);
 }
 
+void layoutmenu(const Arg *arg)
+{
+	/* How many symbols can one layout name fit including the number */
+	int maxlayoutsymbolsize = 10;
+
+	int layoutcount = (sizeof(layouts)/sizeof(*layouts));
```

Comment:
how does this work?

Reply:
It gets the size of the layouts array. maxlayoutsymbolsize is just a size for buffers used in this function